iOS Example Frame/iOS Example FrameUITests/iOS_Example_FrameUITests.swift

```diff
@@ -33,7 +33,7 @@ class iOS_Example_FrameUITests: XCTestCase {
         // Use XCTAssert and related functions to verify your tests produce the correct results.
         
         let app = XCUIApplication()
-        app.buttons["Go to payment page"].tap()
+        app.buttons["Old"].tap()
```

Comment:
I think we can get a better name instead of `Old` @patrick-hoban-cko any suggestions for existing UI. May be have theme names.